docs/book/common_code/images/sequence_cluster_deletion.drawio

```diff
@@ -0,0 +1 @@
+<mxfile modified="2019-03-19T22:53:04.183Z" host="www.draw.io" agent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.121 Safari/537.36" etag="vz0STIlV02B6AmKdRryB" version="10.5.0" type="google"><diagram id="pb2xbBvnu5Ti9wiCyMtv" name="Page-1">7VpNc5swEP01nmkPyQACxxwT56MzbSdpc8hMbjLIRo1ARAh/9NdXMgIbZBts49gd5xR4klbSvsfuSnEH9MPpA4Nx8JP6iHQsw592wG3HskzT7Yk/EpnlCAAZMmLYV9gCeMZ/kQINhabYR0mpI6eUcByXQY9GEfJ4CYOM0Um525CS8qwxHCENePYg0dEX7PMgQ3uOscC/ITwK8plNQ7WEMO+sgCSAPp0sQeCuA/qMUp49hdM+ItJ7uV+ycfdrWouFMRTxJgMeH5/G/fDx2gXD+HU4fDN/vZsXysoYklRt+C0dCD8StWY+yx0hlh/LxzQkP/AQERyJt5sYMRwijphoIQp+WmA3kwBz9BxDTw6dCI0ILOAhEW+meBSscSiGsOKdEBgneDCf1RAIQ17KEjxGv1GSiUOiNOVypn5BugR1h+S7Q4yj6RKkHPSAqFgnm4kuqtVWXCm19tTrZMF8wW+wxLqdj4NKbaPC8oIQ8aA42YIfS+enl8iJYpwgJjZ2njyZ3VMjCmhEeSRNpHPnsYkz4bFzZcs2jsjW7/EQXkwNDrD5eu+yQfISfr/oamzdIiKc++WrxtCSX+WOsUgN1wSPIoENKOc0FA0o8q9lrpEYod7bSrdtkg2HbIQ29XOyfsgvZSvd40suzb3MEIFcUF1OjSs8qqw9USyWXJDnlLkDVoWThKbMQ2rQcvqp2DFBjaHMB5qhOb/FDnf/QO0NHygd/JGCP8uP07FOLZQ6G5iCHk8hp2caSLsnV59caVz580Aqs15G2uUMhno12VpUVT7MolB9oVsbaK2jBVrTLJNb1DhbR1q3xtCaSCucDWdL3WLZIdmw4DXzLPSUWWw1jPc0wUk+jy4vq6G83KPJy6rkX8vYUV7gqsZQS/IqNl6Z56DycjV5vadIPJ5luqkycPx0Y+pVHEM8ZdG+EYDGSDTd+DAJkK/GiWZ1H9XbKzY4DWMDaBgb9gwE3crZ2d41EFyZNYZaCgTdyhEkl9e6dTnuxv6HCRymfqQ8fWU2zVrmxyjTqgjKqsaRxhVQ5cKh6Vlz6wpozeG4LaWtvLvQr2wTWQEZ88Ib00gSKjJOwmEY7yu+/Qsjt6HE7IYSa78wAnZLqrOdGkMtqU7E2Utn5UwH1Z2l6S6Nfcj12uiz+NYjW+WSpbC7dfFt1Bhqq/iuarn3AQrTi7uTONs1vaM9orzcmnzXWF5VQ85h5KXN8xGJU79XDKD8X9oQR5CIckocu+7/H7UdL2HalWMh2DVhOt0aQy2pzalkeNDyRdVKtek3owyFdIxKgvtUW73aqsdVtyW1aYbaKs9cszzPfqlTvC5+I5J1X/zUBtz9Aw==</diagram></mxfile>
```

Comment:
`set deletion timestamp` should probably be initiated from `k8s apiserver` rather than `queue`

Reply:
good point